tests/stratis_min.rs

```diff
@@ -214,6 +214,63 @@ fn test_stratis_min_pool_rename() {
     test_with_stratisd_min_sim(stratis_min_pool_rename);
 }
 
+fn stratis_min_fs_rename() {
+    stratis_min_create_pool_and_fs();
+    let mut cmd = Command::cargo_bin("stratis-min").unwrap();
+    cmd.arg("filesystem").arg("rename").arg("pn").arg("fn").arg("fm");
+    cmd.assert().success();
+    let mut cmd = Command::cargo_bin("stratis-min").unwrap();
+    cmd.arg("filesystem")
+        .assert()
+        .success()
+        .stdout(predicate::str::contains("fm"));
+}
+
+#[test]
+fn test_stratis_min_fs_rename() {
+    test_with_stratisd_min_sim(stratis_min_fs_rename);
+}
+
+fn stratis_min_pool_stop_name() {
+    stratis_min_create_pool_and_fs();
+    let mut cmd = Command::cargo_bin("stratis-min").unwrap();
+    cmd.arg("pool").arg("stop").arg("--name").arg("pn");
+    cmd.assert().success();
+}
+
+#[test]
+// Test stopping a pool using a valid name.
+fn test_stratis_min_pool_stop_name() {
+    test_with_stratisd_min_sim(stratis_min_pool_stop_name);
+}
+
+#[test]
+// Test stopping a pool using an invalid name, that will fail with a
+// "Uuid error".
+fn test_stratis_min_pool_stop_invalid_name() {
```

Comment:
Best to move this one test above the "tests with sim engine" comment. Otherwise, all the tests look good as far as I can tell.

Reply:
OK; resolved.